shared/data/src/main/java/com/ivy/data/repository/mapper/TransactionMapper.kt

```diff
@@ -67,8 +67,8 @@ class TransactionMapper @Inject constructor() {
             TransactionType.EXPENSE -> {
                 Expense(
                     id = TransactionId(id),
-                    title = title?.let { NotBlankTrimmedString.from(it).bind() },
-                    description = description?.let { NotBlankTrimmedString.from(it).bind() },
+                    title = title?.let { NotBlankTrimmedString.from(it).getOrNull() },
```

Comment:
Can we extract title and description as a variables so this code isn't duplicated 3 times?

Reply:
Done